### PR TITLE
EUI-2062-deselect-a-case-master:add deselect a case button

### PR DESCRIPTION
**Pull request by rayliangatos**
Resolves # . (This is applicable only if this pull request relates to a GitHub issue, delete the line otherwise)

Notes:
*
*
*

### CODE REVIEW

File: projects/exui-common-lib/src/lib/components/selected-case/selected-case.component.html [hunk 1]
```diff
@@ -1,32 +1,39 @@
...
+        <th id="th-action" class="govuk-table__header" scope="col">Actions</th>
       </tr>
       </thead>
       <tbody class="govuk-table__body">
-      <tr class="govuk-table__row" *ngFor="let user of case.sharedWith; trackBy: trackByUserId">
-        <td class="govuk-table__cell">{{ user.firstName + ' ' + user.lastName }}</td>
-        <td class="govuk-table__cell">{{ user.email }}</td>
-        <td class="govuk-table__cell"><a href="#"></a></td>
+      <tr class="govuk-table__row" *ngFor="let user of case.sharedWith; index as idx; trackBy: trackByUserId">
+        <td id="user-full-name-{{ idx }}" class="govuk-table__cell">{{ user.firstName + ' ' + user.lastName }}</td>
```
Comment (udaydenduluri33) [inline projects/exui-common-lib/src/lib/components/selected-case/selected-case.component.html:34]: @rayliangatos  I suggest we dont need this id attribute. You could do it with class.

Comment (rayliangatos) [reply projects/exui-common-lib/src/lib/components/selected-case/selected-case.component.html:34]: class can be duplicated, and it's not resilient, use id is a better practice.

Comment (udaydenduluri33) [reply projects/exui-common-lib/src/lib/components/selected-case/selected-case.component.html:34]: Agree to disagree

File: projects/exui-common-lib/src/lib/components/selected-case/selected-case.component.html [hunk 2]
```diff
@@ -1,32 +1,39 @@
...
-    <h2 class="govuk-accordion__section-heading">
-      <button type="button" id="accordion-with-summary-sections-heading-1"
+<div id="govuk-accordion__section-{{ case.caseId }}" class="govuk-accordion__section">
+  <div class="govuk-grid-row govuk-case-header">
+    <div class="govuk-grid-column-three-quarters">
+      <h3 id="case-title" class="govuk-case-title">{{ case.caseTitle }}</h3>
+      <h1 id="case-id" class="govuk-case-sub-title">{{ case.caseId }}</h1>
+    </div>
+    <div class="govuk-grid-column-twenty-percent">
+      <button id="btn-deselect-case" class="govuk-button hmcts-button--secondary" (click) = "onDeselect(case)">Deselect case</button>
```
Comment (ronaldmansveld) [inline projects/exui-common-lib/src/lib/components/selected-case/selected-case.component.html:8]: no spaces around the = sign for attributes

**Pull request merged**